apps/api-server/package.json

```diff
@@ -54,6 +54,7 @@
     "ws": "^7.5.7"
   },
   "devDependencies": {
+    "@types/cron": "^2.4.0",
```

Comment:
Is deze nodig? Volgens mij gebruikt de api-server geen Typescript?

Reply:
Nee, deze was inderdaad niet nodig. Het was tijdelijk toegevoegd terwijl ik nog op zoek was naar een oplossing; ik zal deze er bij de volgende pull request eruit werken.